### DISCUSSION TITLE
Search missing constant/key

### DISCUSSION THREAD

**Discussion by Sepremex** [MohmmedAshraf/laravel-translations#35] [Ideas]
Is there a way to search for missing keys?, I 'm about to use this package on a live project, with about 50 view files, is there a way to search on this files for all "unique" keys that might be missing for translation?
say I have __('Send') in 10 files and in 3/10 files I have __('Sent') and I missed the key "Sent" I think this could be a nice feature for new installation on a production apps or just to make sure we are not missing any keys.

**Comment by MohmmedAshraf** [comment]
Hi @Sepremex please check #38
